go.mod

```diff
@@ -247,7 +247,7 @@ replace (
 	// use cosmos fork of keyring
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
 	// use Cosmos-SDK fork to enable Ledger functionality
-	github.com/cosmos/cosmos-sdk => github.com/evmos/cosmos-sdk v0.47.5-evmos
+	github.com/cosmos/cosmos-sdk => github.com/evmos/cosmos-sdk v0.47.5-evmos.0.20231027081840-e48d219220db
```

Comment:
we need to bump this fork version too

Reply:
you want to release this version of Cosmos SDK as a separate release on our fork?